.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
+node_js: stable
+script:
+- npm run test:lint:js
+- npm run test:lint:wc

```

Comment:
Since your `npm test` script just calls these two, I think you can exclude the `script:` block entirely, or have it run `npm test`.